client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionsTab.tsx

```diff
@@ -71,6 +71,7 @@ const InteractionsTab: React.FC<Props> = (props: Props): JSX.Element => {
                 {
                     getDatesToInvestigate(doesHaveSymptoms, symptomsStartDate, coronaTestDate).reverse().map(date =>
                         <ContactDateCard
+                            allInteractions = {interactions}
```

Comment:
look at the context comment

Reply:
maybe after solving the bugs we should change it as well